### ISSUE TITLE
text loss problem

### ISSUE THREAD

**Issue by jmwang0117** [LatentActionPretraining/LAPA#16]
Thank you for your excellent work!

When I fine-tune, I find that the action loss can be reduced normally, but the text loss is above 10 and the text acc is around 0.5. Is this normal? In order to achieve the effect and performance in the paper, do I need to continue training to minimize the text loss?

**What are the optimal action loss and text loss?**

![image](https://github.com/user-attachments/assets/6fed5db3-28af-45d5-b22c-9835e9a67a45)

**Comment by seonghyeonye** [comment]
Text loss is not important during fine-tuning because we do not use text during evaluation. Instead, action accuracy is important and we fine-tuned the model until the training action accuracy is ~95%.
